platform/innovium/sonic-platform-modules-cameo/esc601-32q/modules/mcp3422.c

```diff
@@ -0,0 +1,429 @@
+/*
+ * mcp3422.c - driver for the Microchip mcp3421/2/3/4/5/6/7/8 chip family
```

Comment:
can you use linux kernel driver?

Reply:
if so , I need to modify  kenerl default config.  "# CONFIG_MCP3422 is not set"  is default value in current kernel config.

Reply:
@lguohan any advice?

Reply:
yes, can you submit pr to enable the kernel driver?